index.bs

```diff
@@ -9844,19 +9849,134 @@ Constructors</h5>
 <dl dfn-type=constructor dfn-for="AudioWorkletNode">
 	: <dfn>AudioWorkletNode(context, name, options)</dfn>
 	::
-		Let <var>node</var> be a new {{AudioWorkletNode}} object.
-		<a href="#audionode-constructor-init">Initialize</a>
-		<var>node</var>. Perform the <a href="#instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
-		construction procedure</a> of an
-		{{AudioWorkletNode}} and the corresponding
-		{{AudioWorkletProcessor}} object. Return
-		<var>node</var>.
+		When the constructor of {{AudioWorkletNode}} is invoked in the
+		main global scope, the following construction algorithm MUST be
+		performed.
+
+		During the construction of an AudioWorkletNode, a
+		corresponding {{AudioWorkletProcessor}} instance is also
+		automatically created in the {{AudioWorkletGlobalScope}}.
+		Note that the instantiation of these object pairs spans the
+		control thread and the rendering thread.
```

Comment:
Maybe this is step 0 of the algorithm below?

Reply:
It's already there at L.10198, but I wanted to emphasize the procedure because it is unconventional.